debugpy_info.json

```
@@ -0,0 +1,37 @@
+{
+    "macOS": {
+        "url": "https://files.pythonhosted.org/packages/bd/a3/5e37ce13c7dd850b72a52be544a058ed49606ebbbf8b95b2ba3c1db5620a/debugpy-1.7.0-cp311-cp311-macosx_11_0_universal2.whl",
+        "hash": [
+            "sha256",
+            "538765a41198aa88cc089295b39c7322dd598f9ef1d52eaae12145c63bf9430a"
+        ]
```

Maybe make this a dict? That way you disambiguate what's the hash algorithm and what's the hash value compared to having to memorize what item 1 and what item 2 are.

```suggestion
        "hash": {
            "sha256":
            "538765a41198aa88cc089295b39c7322dd598f9ef1d52eaae12145c63bf9430a"
        }
```